Var/src/main.c

```diff
@@ -0,0 +1,38 @@
+#include "var.h"
+
+int main(){
+	Var dynamics[10] = {0};
+	size_t length = sizeof(dynamics)/sizeof(*dynamics);
```

Comment:
So the compiler calculates the value at compile time?

Reply:
No, but the `10` you have now is a magic number and doesn't really mean anything. If it was a constant it would be easier to change it later and the `10` would get a meaning (ARRAY_SIZE? ARRAY_LENGTH...)